nomad/poc-2.hcl

```diff
@@ -24,125 +40,104 @@ job "poc-2" {
 
             service {
                 provider = "nomad"
-                name = "holochainBootstrap"
-                port = "holochainBootstrap"
+                name = "holochainTestRemoteV1"
+                port = "holochainTestRemoteV1"
+
+                // check {
+                //     name     = "remote_v1_server_up"
+                //     type     = "tcp"
+                //     port     = "holochainTestRemoteV1"
+                //     interval = "10s"
+                //     timeout  = "2s"
+                // }
             }
 
-            service {
-                provider = "nomad"
-                name = "holochainSignal"
-                port = "holochainSignal"
+            template {
+                data = file("nomad/poc-2/init.sh")
+                destination = "local/init.sh"
+                perms = "555"
             }
 
             driver = "raw_exec"
             config {
                 command = "/usr/bin/env"
-                args = ["bash", "-c", 
-                <<ENDOFSCRIPT
-                set -xe
-                env
-
-                # git ls-remote https://github.com/holochain/holochain.git --branch develop
-
-                nix shell -vL github:holochain/holochain#holochain \
-                    --override-input versions 'github:holochain/holochain?dir=versions/weekly' \
-                    --override-input versions/holochain 'github:holochain/holochain/a585a619d68bd47d2e995a773cdf76dea08fdca3' \
-                    --command \
-                    hc-run-local-services \
-                    --help
-
-                export RUST_LOG=debug
-                nix shell -vL github:holochain/holochain#holochain \
-                    --override-input versions 'github:holochain/holochain?dir=versions/weekly' \
-                    --override-input versions/holochain 'github:holochain/holochain/a585a619d68bd47d2e995a773cdf76dea08fdca3' \
-                    --command \
-                    hc-run-local-services \
-                        --bootstrap-interface 0.0.0.0 \
-                        --bootstrap-port ''${NOMAD_PORT_holochainBootstrap} \
-                        --signal-interfaces 0.0.0.0 \
-                        --signal-port ''${NOMAD_PORT_holochainSignal}
-                ENDOFSCRIPT
+                args = ["bash", "-c",
+                    <<ENDOFSCRIPT
+                    set -xeu
+
+                    source local/init.sh
```

Comment:
The only exported variable from `init.sh` is `JOB_STATE_DIRECTORY` which isn't used here. Just run it instead of sourcing?

Reply:
`PWD` is the reason i chose `source` in the first place so that the rest of the code can expect to be in the holochain directory

Reply:
The comment hasn't gone where the question was raised, it would be useful to have it in both places